api/src/main/java/marquez/api/DatasetResource.java

```diff
@@ -150,6 +151,28 @@ public Response list(
     return Response.ok(new ResultsPage<>("datasets", datasets, totalCount)).build();
   }
 
+  @Timed
+  @ResponseMetered
+  @ExceptionMetered
+  @DELETE
+  @Path("{dataset}")
+  @Produces(APPLICATION_JSON)
+  public Response deleteDataset(
```

Comment:
`DatasourceResource.delete` would be enough I think.

Reply:
Changed.